src/Runtime/OMInstrument.inc

```diff
@@ -63,29 +72,70 @@ void TimeInit() {
 #endif
 
 #ifdef _WIN32
-void ReportTime() {}
+inline void WinTimerSub(LARGE_INTEGER newTime, LARGE_INTEGER prevTime,
```

Comment:
There is a similar need for times in the utils/RunONNXLib.cpp. My fault in that I did not try to unify the interface, here is what I used there
```
#ifdef _WIN32
inline void processStartTime() {}
#else
inline void processStartTime() {
  if (!measureExecTime)
    return;
  gettimeofday(&startTime, NULL);
}
#endif

// Process stop time, store result in log.
#ifdef _WIN32
inline void processStopTime() {}
#else
inline void processStopTime() {
  if (!measureExecTime)
    return;
  gettimeofday(&stopTime, NULL);
  timersub(&stopTime, &startTime, &result);
  uint64_t time =
      (uint64_t)result.tv_sec * 1000000ull + (uint64_t)result.tv_usec;
  timeLogInMicroSec.emplace_back(time);
}
#endif
```
 Maybe I could try to unify the interface here and in that file, so that we may simply reuse the time between the two places. It looks like the commonality could be expressed as a simple function whose template is this

```
unsigned long long timer_interval_since_last() {
  static bool first = true;
  // pseudo code
  static XXX lastTime;
  static XXX currTime;
  if (first) {
    first = false;
    lastTime = XXX_time()
    return;
  }
 currTme = XXX_time()
 intervalInMicroSec = XXX_time_diff(lastTime, currTime);
 lastTime = currTime;
 return intervalInMicroSec;
}
```

We could then use this in both places. It simply restrict us to a single timer... probably fine, otherwise we can bring an abstract struct to handle it too.